sw/device/silicon_creator/manuf/BUILD

```diff
@@ -0,0 +1,31 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:opentitan_test.bzl", "cw310_params", "opentitan_functest", "verilator_params")
+
+opentitan_functest(
+    name = "ft",
+    srcs = ["ft.c"],
+    cw310 = cw310_params(
+        bitstream = "//hw/bitstream:rom_otp_dev",
+        # TODO(lowRISC/opentitan#13603): Remove this "manual" tag when the
+        # bitstream target can fetch pre-spliced bitstream from GCP.
+        tags = ["manual"],
```

Comment:
I don't think you need this anymore, we can splice in CI and locally now.

Reply:
Done.